### PR TITLE
[FrameworkBundle] MicroKernelTrait configureRouting(Configurator)

### PR DESCRIPTION
**Pull request by vudaltsov**
https://github.com/symfony/symfony/pull/32937

### CODE REVIEW

File: configuration/front_controllers_and_kernel.rst [hunk 1]
```diff
@@ -85,7 +85,7 @@ you must implement them all:
 :method:`Symfony\\Component\\HttpKernel\\KernelInterface::registerBundles`
     It must return an array of all bundles needed to run the application.
 
-:method:`Symfony\\Bundle\\FrameworkBundle\\Kernel\\MicroKernelTrait::configureRoutes`
+:method:`Symfony\\Bundle\\FrameworkBundle\\Kernel\\MicroKernelTrait::configureRouting`
     It adds individual routes or collections of routes to the application (for
     example loading the routes defined in some config file).
```
Comment (wouterj) [inline configuration/front_controllers_and_kernel.rst]: I think we need a `.. versionadded:: 5.1` here to indicate that this method was introduced in 5.1

File: configuration/micro_kernel_trait.rst [hunk 2]
```diff
@@ -98,9 +98,9 @@ that define your bundles, your services and your routes:
     of what you see in a normal ``config/packages/*`` file). You can also register
     services directly in PHP or load external configuration files (shown below).
 
-**configureRoutes(RouteCollectionBuilder $routes)**
+**configureRouting(RoutingConfigurator $routes)**
     Your job in this method is to add routes to the application. The
-    ``RouteCollectionBuilder`` has methods that make adding routes in PHP more
+    ``RoutingConfigurator`` has methods that make adding routes in PHP more
     fun. You can also load external routing files (shown below).
```
Comment (wouterj) [inline configuration/micro_kernel_trait.rst]: The same here

**Review by wouterj — CHANGES_REQUESTED** [review]
Thanks for submitting the docs for your code feature! I would like to add 2 versionadded indications (it doesn't look relevant for me to put one in the 3rd file)

File: configuration/front_controllers_and_kernel.rst (hunk 1 above)
Comment (vudaltsov) [reply configuration/front_controllers_and_kernel.rst]: Done! Did I get you correctly?

Comment (wouterj) [reply configuration/front_controllers_and_kernel.rst]: Yep perfect!

**Comment by wouterj** [comment]
Hi @vudaltsov! Thanks for submitting a doc PR alonside your code one.

However, when looking at the current code, it seems like the route is still called `configureRoutes()`: https://github.com/symfony/symfony/blob/5.1/src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php#L159-L163 So I'm going to close this PR, as it seems like there is nothing more to do (the micro kernel example has recently been updated to use the configurators).

If I'm wrong, feel free to comment :)

**Pull request closed without merging**